examples/list_slaves.rs

```diff
@@ -0,0 +1,35 @@
+//! EtherCAT master example: List slaves
+//!
+//! Opens master 0 (read-only), queries slave info for positions 0..31 and
+//! prints discovered slaves.
+use ethercat::{Master, SlavePos};
+use std::{io};
+
+pub fn main() -> Result<(), io::Error> {
+    // initialize logging
```

Comment:
I think these kinds of comments are unnecessary. The code should be self-explanatory anyway!